scheduler.rb

```diff
@@ -35,12 +36,25 @@
   Iam.settings.DB[:plugins].each do |p|
     # Require the plugin based on the name in the table
     require_relative "plugins/#{p[:name]}/plugin.rb"
-    # For each key in cache
-    cache.keys.each do |key|
-      # Store the node information in the proper table with the plugin's store
-      # method. The plugin object is retrieved from the name string using
-      # Object.const_get. Do not try to store keys that store a datetime object.
-      Object.const_get(p[:name]).new.store key unless key.end_with?('datetime')
+
+    # if DBSIZE plugin, use the db_cache, otherwise use the regular cache
+    if p[:name] == 'DBSize'
```

Comment:
Making this depend on the name of the plugin seems a little brittle, maybe we should do this by resource? (plugins declare their resource, I believe). Then our caches can correspond to the resource type, a cache for nodes, one for databases, etc.